### DISCUSSION TITLE
mf 的插件为什么生产环境才会使用 top await，开发环境就是直接引入入口文件

### DISCUSSION THREAD

**Discussion by qiugu** [umijs/umi#11930] [Q&A]
mf 的插件会在生产环境下生成asyncEntry的文件，并且入口文件修改为这个文件，开发环境下则是umi.ts 文件，这是为什么

**Comment by fz6m** [comment]
生产是没有 top await 的，开发环境才有。

**Comment by qiugu** [comment]
![image](https://github.com/umijs/umi/assets/33192247/9af5c1e4-eddb-476d-819e-87aaec74794a)
只有生产环境才有top await啊，开发没有，执行build以后.umi文件里面才有asyncEntry这个文件啊

**Reply by fz6m** [reply]
这个文件里没有 `await`
